ajax/libs/react-trend/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "react-trend",
+  "filename": "src/index.min.js",
```

Comment:
`filename` would be the main files with the relative path to `basePath` in auto-update config.